### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by baptisteguidoux**
Update documentation

### CODE REVIEW

File: Documentation~/TableOfContents.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 * [ShotGrid Integration](index.md)
-* [Setup](installPython.md)
+* Setup:
```
Comment (baptisteguidoux) [inline Documentation~/TableOfContents.md:2]: `installPython.md` was deleted in a previous commit

File: Documentation~/index.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-We recommend installing the latest version of Unity 2020 LTS via the Unity Hub; [2019.3](https://unity3d.com/get-unity/download) is the minimum.
+# System Requirements
```
Comment (baptisteguidoux) [inline Documentation~/index.md:1]: Aligned with Python for Unity requirements

File: Documentation~/shotGridConfig.md [hunk 3]
```diff
@@ -8,8 +8,8 @@ with ShotGrid inside of Unity.
 If you are using a custom configuration, it should be straightforward to adapt
 it to also use `tk-unity`. The recommended approach is to compare 
 `tk-unity-config` with its matching `tk-config-default2` version. For example, 
-you could compare `tk-config-unity` v1.2.9.1 with `tk-config-default2` 
-v1.2.9.1 and add the required changes.
+you could compare `tk-config-unity` v1.3.19.1 with `tk-config-default2`
```
Comment (baptisteguidoux) [inline Documentation~/shotGridConfig.md:11]: Update to latest release

File: Documentation~/cleanShotGrid.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+## Uninstall existing ShotGrid Desktop apps
+
+Before installing a newer version of ShotGrid Desktop, it is recommended to remove
+the previously installed, older version, of ShotGrid Desktop.
```
Comment (lassond) [inline Documentation~/cleanShotGrid.md:4]: Replace "the previously installed, older version, of ShotGrid Desktop." with "the previous version"

File: Documentation~/cleanShotGrid.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+To uninstall ShotGrid Desktop:
+
+- On Mac: Applications → Utilities → Uninstall ShotGrid Desktop
+- On Windows: Add/Remove programs → Search for ShotGrid Desktop
+
+## Clear ShotGrid Desktop cache
```
Comment (lassond) [inline Documentation~/cleanShotGrid.md:11]: "Clear ShotGrid Desktop cache" -> "Clear The ShotGrid Desktop Cache"

File: Documentation~/cleanShotGrid.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+If you edit Pipeline Configuration, or a ShotGrid setting, but do not see
+ the effects of this change, try to clear the ShotGrid Desktop cache.
```
Comment (lassond) [inline Documentation~/cleanShotGrid.md:14]: "If you edit a Pipeline Configuration or a ShotGrid setting but do not see
 the effects of this change, try to clear the ShotGrid Desktop cache."

File: Documentation~/index.md (hunk 2 above)
Comment (lassond) [reply Documentation~/index.md:1]: ok, I see where these references to Mac come from. The Python package is supported for Mac but (I think) not the SG package

**Review by lassond — APPROVED** [review]

**Pull request merged**